backend/src/test/java/codezap/member/repository/MemberJpaRepositoryTest.java

```diff
@@ -1,45 +1,139 @@
 package codezap.member.repository;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.jpa.repository.JpaRepository;
 
 import codezap.category.domain.Category;
 import codezap.category.repository.CategoryJpaRepository;
+import codezap.category.repository.CategoryRepository;
 import codezap.fixture.CategoryFixture;
+import codezap.fixture.MemberFixture;
+import codezap.global.exception.CodeZapException;
 import codezap.global.repository.JpaRepositoryTest;
 import codezap.member.domain.Member;
-import codezap.member.fixture.MemberFixture;
 import codezap.template.domain.Template;
 import codezap.template.repository.TemplateJpaRepository;
+import codezap.template.repository.TemplateRepository;
 
 @JpaRepositoryTest
 class MemberJpaRepositoryTest {
 
-    private final MemberJpaRepository memberJpaRepository;
-    private final TemplateJpaRepository templateJpaRepository;
-    private final CategoryJpaRepository categoryJpaRepository;
+    private final MemberRepository memberRepository;
+    private final TemplateRepository templateRepository;
+    private final CategoryRepository categoryRepository;
 
     @Autowired
-    MemberJpaRepositoryTest(MemberJpaRepository memberJpaRepository, TemplateJpaRepository templateJpaRepository,
-            CategoryJpaRepository categoryJpaRepository
+    MemberJpaRepositoryTest(
+            MemberJpaRepository memberRepository,
+            TemplateJpaRepository templateRepository,
+            CategoryJpaRepository categoryRepository
     ) {
-        this.memberJpaRepository = memberJpaRepository;
-        this.templateJpaRepository = templateJpaRepository;
-        this.categoryJpaRepository = categoryJpaRepository;
+        this.memberRepository = memberRepository;
+        this.templateRepository = templateRepository;
+        this.categoryRepository = categoryRepository;
     }
 
-    @Test
-    void fetchByTemplateId() {
-        // Then
-        Member member = ((JpaRepository<Member, Long>)memberJpaRepository).save(MemberFixture.memberFixture());
-        Category category = ((JpaRepository<Category, Long>)categoryJpaRepository).save(CategoryFixture.getFirstCategory());
-        Template template = new Template(member, "title", "description", category);
-        Template savedTemplate = ((JpaRepository<Template, Long>)templateJpaRepository).save(template);
+    @Nested
+    @DisplayName("id로 Member 조회")
+    class fetchById {
+        @Test
+        @DisplayName("성공 : id로 Member를 알아낼 수 있다.")
+        void fetchByIdSuccess() {
+            Member member = MemberFixture.getFirstMember();
+            member = memberRepository.save(member);
```

Comment:
```suggestion
            Member member = memberRepository.save(MemberFixture.getFirstMember());
```
[제안]
정말 사소한데, `MemberFixture.getFirstMember()` 를 변수로 추출할 필요는 없을 것 같아요. 
member 변수에 값을 재할당 할 필요 없이 `MemberRepository.save()` 로 저장 한 반환값을 필드로 추출하는 건 어떤가요?

(어떤 방법도 타당하게 느껴져서, 켬미의 선호에 맞게 선택해 주시면 좋을 것 같아요!)

Reply:
```suggestion
            Member member = MemberFixture.getFirstMember();
            Member saved = memberRepository.save(member);
```
(짱수 코멘트 보고 작성합니다)

저는 이런 방식을 더 선호해요. 

짱수와 달리 저는 가능하다면 호출된 메서드의 반환값을 명시하기 위해 변수를 할당하는 게 좋다고 생각합니다~

Reply:
저는 짱수의 의견에 동의해서  MemberFixture.getFirstMember() 를 변수로 추출하지 않는 방향으로 수정하였습니다 ~